packages/mds-event-processor/src/geo/geo.ts

```diff
@@ -0,0 +1,72 @@
+let service_areas = require('./service-areas')
+let turf_main = require('@turf/helpers')
+let turf = require('@turf/boolean-point-in-polygon')
+
+const log = require('loglevel')
+
+let district_areas = Array()
+for (let index in service_areas['features']) {
+  let district_uuid = service_areas['features'][index]['properties']['dist_uuid']
+  let area = service_areas['features'][index]['geometry']['coordinates']
+  // still servicing
+  district_areas[district_uuid] = service_areas['features'][index] // turf_main.polygon(area[0]));
+}
+
+let findServiceAreas = function(lng: any, lat: any) {
+  let areas = []
+  let turf_pt = turf_main.point([lng, lat])
+  for (let key in district_areas) {
+    if (turf.default(turf_pt, district_areas[key])) {
+      areas.push({ id: key, type: 'district' })
+    }
+  }
+  return areas
+}
+
+let moved = function(first_data: any, second_data: any) {
+  let limit = 0.00001 // arbitrary amount
+  let lat_diff = Math.abs(first_data.latitude - second_data.latitude)
+  let lng_diff = Math.abs(first_data.longitude - second_data.longitude)
+  return lng_diff > limit || lat_diff > limit // very computational efficient basic check (better than sqrts & trig)
+}
+
+// Helper funtion to calculate distance between two points given latitudes and longitudes
+// Unit is default miles but can be expressed in kilometers given the value 'K'
+function gpsDistance(lat1: number, lon1: number, lat2: number, lon2: number, unit: string) {
```

Comment:
There's a more precise version of this function that takes into the account the curvature of the earth's surface called `routeDistance` in `mds-utils`.